cmd/copy.go

```diff
@@ -302,6 +303,11 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 		return cooked, err
 	}
 
+	// Do not check for an error on this. Leaving this as true doesn't hurt, as the event never triggers unless it's an s2s copy anyway.
+	// This is thanks to a type assertion attempt.
+	// Atop this, leaving this on board for up/downloads in the future would be useful.
```

Comment:
Good point. Do you think we should consider enabling it right now for all transfers, not just S2S?

Reply:
I'm ok with it, up to @adreed-msft